src/common/snippets/include/snippets/lowered/loop_manager.hpp

```diff
@@ -100,37 +100,83 @@ class LinearIR::LoopManager {
     };
     using LoopInfoPtr = std::shared_ptr<LoopInfo>;
 
+    /**
+     * @brief Clone Loop Manager with new expressions
+     * @param expr_map map of new and old expressions
+     * @return the copy
+     */
     std::shared_ptr<LoopManager> clone_with_new_expr(const ExressionMap& expr_map) const;
-    size_t add_loop_info(const LoopInfoPtr& loop);
-    void remove_loop_info(size_t index);
+
+    /**
+     * @brief Get target Loop Info
+     * @param index loop ID
+     * @return the LoopInfo shared ptr
+     */
     LoopInfoPtr get_loop_info(size_t index) const;
+    /**
+     * @brief Get count of loops
+     * @return count of loops in the map
+     */
     size_t get_loop_count() const { return m_map.size(); }
+    /**
+     * @brief Get loop map [loop_id -> loop info]
+     * @return loop map
+     */
     const std::map<size_t, LoopInfoPtr>& get_map() const;
-
-    // Return outer Loop IDs
+    /**
+     * @brief Get loop IDs of expression that are outer (upper) than `loop_id`
+     * @param expr the target expression
+     * @param loop_id the target loop ID
+     * @return vector out outer loop IDs
+     */
     static std::vector<size_t> get_outer_expr_loops(const ExpressionPtr& expr, size_t loop_id);
-
+    /**
+     * @brief Create new LoopInfo and mark all expressions in loop bounds by new loop ID
+     * @param loop_begin_pos the first expression iterator
+     * @param loop_end_pos the next iterator after last expression
+     * @param loop_depth count of loops for marking
+     * @param vector_size increment of loop
+     */
     void mark_loop(LinearIR::constExprIt loop_begin_pos,
                    LinearIR::constExprIt loop_end_pos,
                    size_t loop_depth, size_t vector_size);
-    // Return Loop ID
+    /**
+     * @brief Create new LoopInfo and mark all expressions in loop bounds by new loop ID with more manual parameters
+     * @param loop_begin_pos the first expression iterator
+     * @param loop_end_pos the next iterator after last expression
+     * @param work_amount work amount of the loop
+     * @param increment increment of the loop
```

Comment:
There are two type of increments in the loop: work amount increment and pointer increment.
I think we need to explicitly explain the increment meaning here, since you renamed the `work_amount` argument below.